app/controllers/community_resources_controller.rb

```diff
@@ -62,13 +53,14 @@ def community_resource_params
           :facebook_url,
           :is_approved,
           :name,
-          :organization_id,
```

Comment:
was on the fence here if this gets removed due to organization_attributes, or if both are needed ?